examples/flutter_gallery/lib/main.dart

```diff
@@ -14,8 +14,9 @@ import 'gallery/app.dart';
 
 // Sets a platform override for desktop to avoid exceptions. See
 // https://flutter.dev/desktop#target-platform-override for more info.
+// TODO(gspencergoog): Remove once TargetPlatform includes all desktop platforms.
 void _enablePlatformOverrideForDesktop() {
-  if (!kIsWeb && (Platform.isMacOS || Platform.isWindows || Platform.isLinux)) {
+  if (!kIsWeb && (Platform.isWindows || Platform.isLinux)) {
```

Comment:
I think this is the only copy of this fragment that might be copied by developers.  It might be helpful to explain that we're doing it to enable running gallery tests.

Reply:
OK, done.